grafana/src/shared.libsonnet

```diff
@@ -10,7 +10,9 @@ local layout = import './layout.libsonnet';
 local prometheusNamespace = std.extVar('prometheusNamespace');
 local cortexDataSource = std.extVar('cortexDataSource');
 local instanceFilter = 'namespace="$namespace",service=~"$service.*"';
-local namespaceFilter = 'namespace="$namespace",app_name=~"$adapter.*"';
+local appFilter = 'namespace="$namespace",app_name=~"$adapter.*"';
+local filterType = std.extVar('filterType');
+local masterFilter = if filterType == "app" then appFilter else instanceFilter;
```

Comment:
How do you feel about naming this something like `eaSelector`? Seems like a more accurate name, since the set of labels contained in the variable should narrow down to one and only one external adapter's metrics

Reply:
Sounds good to me